ops/README.md

```diff
@@ -0,0 +1,124 @@
+NEAR application layer running with Tendermint consensus.
+
+## Running locally
+
+To run NEAR locally you would need docker, see [installation instructions](https://www.docker.com/get-started).
+Then run the following:
+```bash
+./ops/deploy_local.sh
+```
+
+After it starts you can open studio at [http://localhost](http://localhost).
+
+To tear it down run:
+```bash
+./ops/teardown_local.sh
+```
+
+## Running remotely
+Similarly you deploy the network to the GCloud:
+```bash
+./ops/deploy_remote.sh
+```
+When the network is deployed it will print the address of the studio.
+
+## Developing locally
+Developing from the source code requires more installation and configuration.
+
+### Installation
+
+This mode of running requires compilation of the Rust code and installed Tendermint.
```

Comment:
may want to clarify that this is not required for docker or cloud?

Reply:
I changed the title of the section and added clarification to make it less confusing.